R/PipeOpImputeHist.R

```diff
@@ -60,7 +60,7 @@ PipeOpImputeHist = R6Class("PipeOpImputeHist",
   ),
   private = list(
 
-    .select_cols = function(task) task$feature_types[get("type") %in% c("numeric", "integer"), get("id")],
+    .select_cols = function(task) selector_type(c("numeric", "integer"))(task),
```

Comment:
I *think* you could jsut do `.select_cols = selector_type(c("numeric", "integer"))` here, but I'm not 100% sure. Have you tried that and it failed for some reason?

Reply:
So I have not fully understood why, but it fails.

What happens is, that the `types` supplied to `slector_type` are lost after construction.
and I get `Object 'types' not found.`

Reply:
interesting, probably R6 takes away environments of methods

Reply:
https://github.com/r-lib/R6/issues/80